Badge.md

```diff
@@ -1,3 +1,65 @@
 (Version >= 1.16.0)
 
-Please read: https://github.com/louislam/uptime-kuma/pull/1119
\ No newline at end of file
+- [Description](#description)
+- [Status badge](#status-badge)
+- [Ping and uptime badge](#ping-and-uptime-badge)
+- [Screenshots](#screenshots)
+
+
+# Description
+
+Status, ping and uptime badges, generated in node-js, no internet access required (based on [badge-maker](https://www.npmjs.com/package/badge-maker) aka. [shields.io](http://shields.io/))
+
+There are three API endpoints that return an SVG graphic for a given (public) monitor:
+
+**Status** `<kuma-url>/api/badge/:monitorID/status` e.g. `<kuma-url>/api/badge/1/status` returning sth like: ![image](img/badge/148052351-08a5713f-2bdf-4034-ab2a-29c7638cb25c.png)
+
+**Ping** `<kuma-url>/api/badge/:monitorID/ping/:duration` e.g. `<kuma-url>/api/badge/1/ping` returning sth like: ![image](img/badge/148052406-55b55a76-f61b-4a2a-a036-53b557eebc49.png)
+
+**Uptime** `<kuma-url>/api/badge/:monitorID/uptime/:duration` e.g. `<kuma-url>/api/badge/1/uptime/24` returning sth like: ![image](img/badge/147946965-5a706c7f-1efe-4056-b06b-b149904f87b5.png)
+
+# Status badge
```

Comment:
I think we can skip these explainations and instead show the badge-maker. Currently, there are a few badges which are not included in this explaination like cert-expiry

Said feature is hidden here:
![image](https://github.com/louislam/uptime-kuma-wiki/assets/26258709/ec3267ce-6a2f-4653-bf51-3553be119b5e)
![image](https://github.com/louislam/uptime-kuma-wiki/assets/26258709/5dd6e68a-81cb-4358-b4cc-8f7633bc12a1)
![image](https://github.com/louislam/uptime-kuma-wiki/assets/26258709/67a16e8e-bc49-44af-82c2-ef0da83e5955)